.github/actions/spell-check/patterns.txt

```diff
@@ -0,0 +1,5 @@
+(?:0[Xx]|U\+|#)[a-f0-9A-FGgRr]{2,}[Uu]?[Ll]{0,2}\b
+addNSECRecordToLW.*DNSName.*powerdnt.com.*QType::NS.*res->d_records
+data:[a-zA-Z=;,/0-9+]+
+BOOST_CHECK_EQUAL\(b64, "[a-zA-Z=;,/0-9+]+"
+\b([A-Za-z])\1{3,}\b
```

Comment:
The `\b`s are here because otherwise `aaarec` would be accepted when it's a typo for `aaaarec`